### PR TITLE
header

### PR DESCRIPTION
**Pull request by hieunguyen64**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+<body>
+    <div class="container">
+        <div class="row">
+            <div class="logo col-lg-2 col-sm-6 ">
+                <img src="image/logo.jpeg" width="150px">
+            </div>
+            <div class="search-container col-lg-7 col-sm-12">
+                <form class="searchbar">
+                    <input type="text" placeholder="Search.." name="search">
```
Comment (thienhophu) [inline index.html]: Search.. => Search...

File: index.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+                    <button type="submit"><i class="fa fa-search"></i></button>
```
Comment (thienhophu) [inline index.html]: press search shows alert with "Coming soon..." message

File: index.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    <script src="js/jquery-3.4.1.slim.min.js"></script>
+    <script src="js/popper.min.js"></script>
+    <script src="js/bootstrap.js"></script>
+</head>
+
+<body>
+    <div class="container">
+        <div class="row">
+            <div class="logo col-lg-2 col-sm-6 ">
+                <img src="image/logo.jpeg" width="150px">
```
Comment (thienhophu) [inline index.html]: height and how the image should be rendered

File: index.html [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+            </div>
+            <div class="search-container col-lg-7 col-sm-12">
+                <form class="searchbar">
+                    <input type="text" placeholder="Search.." name="search">
+                    <button type="submit"><i class="fa fa-search"></i></button>
+                </form>
+            </div>
+            <div class="addtocart col-lg-3 col-sm-6 ">
+                <form class="shopping-cart">
+                    <button type="button"><i class="fa fa-shopping-cart"></i></button>
```
Comment (thienhophu) [inline index.html]: alert with "Coming soon..." message

File: index.html (hunk 1 above)
Comment (hieunguyen64) [reply index.html]: done

File: index.html (hunk 2 above)
Comment (hieunguyen64) [reply index.html]: done

**Comment by thienhophu** [comment]
Go to go

**Review by thienhophu — APPROVED** [review]

**Pull request merged**